libyul/optimiser/ExpressionInliner.cpp

```diff
@@ -23,10 +23,10 @@
 #include <libyul/optimiser/InlinableExpressionFunctionFinder.h>
 #include <libyul/optimiser/Substitution.h>
 #include <libyul/optimiser/Semantics.h>
+#include <libyul/optimiser/NameCollector.h>
+#include <libyul/optimiser/Metrics.h>
```

Comment:
Could you please re-order the includes alphabetically?

Reply:
ok! done